index.html

```diff
@@ -1,139 +1,90 @@
-            <div class="box-horizontal"></div>
-            <div id="about-us"><h1><a href="#about">About us</a></h1></div>
-        </div>
+      <div class="landing-section-middle">
+        <div class="box-vertical"></div>
+        <div class="box-vertical"></div>
+      </div>
 
-        <div class="box-container">
-            <div class="box-vertical"></div>
-            <div class="box-vertical"></div>
-        </div>
+      <div class="landing-section-footer">
+        <a href="#carss">Cars</a>
+        <div class="box-horizontal"></div>
+        <a href="#contact">Sign up</a>
+      </div>
+    </section>
 
-        <div class="container-2-pc">
-            <div id="cars"><h1><a href="#carss">Cars</a></h1></div>
-            <div class="box-horizontal" id="box-horizontal-2"></div>
-            <div id="sign-up"><h1><a href="#contact">Sign up</a></h1></div>
-        </div>
-        <!--Mobile-->
-        <div class="mobile">
-            <div class="mobile-heading">
-                <h1 id="mobile">Driver’s Club.</h1>
-            </div>
-            <div class="mobile-about">
-                <h1>About us</h1>
-            </div>
-            <div class="mobile-cars">
-                <h1>Cars</h1>
-            </div>
-            <div class="mobile-sign-up">
-                <h1>Sign up</h1>
-            </div>
-        </div>
+    <section class="about-section" id="about">
+      <h1>Who are we?</h1>
+      <div class="about-section-text-container">
+        <p>
+          Greetings from the minds behind the movement! We are Johnny Jones and
+          Will Carlson, the co-founders of Driver’s Club. Born from a shared
+          vision of a more connected and compassionate Europe, we've been
+          steering the wheel towards positive change through our unique rallies.
+        </p>
+        <p>
+          Our journey began with a mutual belief in the power of collective
+          action. As seasoned travelers and advocates, we recognized the
+          potential of rallying communities across Europe to amplify our impact.
+          Driver’s Club emerged as our vessel to turn this vision into a
+          reality.
+        </p>
+      </div>
+      <div class="image-container">
+        <img src="images/getleman.jpg" />
+        <img src="images/alfonso-castro-cGqg8Bx2Kbw-unsplash.jpg" />
+      </div>
+    </section>
 
-    </div>
- 
-    <!--About section-->
-    <div class="about-section" id="about">
-        <div id="heading">
-            <h1>Who are we?</h1>
-        </div>
-        <div class="text-container">
-            
-            <div class="paragraph">
-                <p>Greetings from the minds behind the movement! We are Johnny Jones and Will Carlson, the co-founders of Driver’s Club. Born from a shared vision of a more connected and compassionate Europe, we've been steering the wheel towards positive change through our unique rallies.</p>
-            </div>
```

Comment:
put the calss on the p tag, no need for div. More divs, more stuff for the browser to interpret. Don't make it hard for him.